### ISSUE TITLE
Player Attendance showing all player queued rather than confirmed player

### ISSUE THREAD

**Issue by LeonGeb** [arnecls/raidplaner#147]
Hi Arne,

I'm not sure if this is a bug but when I queue for a raid i'm counted as 1/8 Player already. This could be irritating for some people.

Is there a way to change it so only confirmed player will be counted or could this be changed in future release?

**Comment by arnecls** [comment]
The number is meant to be a total attendance count rather than a count of set-up players.
Let me explain:

As a raidlead (and sometimes as a player, too) you want to know if - at least in theory - a raid will take place or not.
I agree that some people tend to say "oh there are already 10/10 players attended, I'll skip this raid" but if you think about this, this is actually not a wise choice (100% chance NOT to be taken chosen over a low chance to be taken). BUT I also have seen players that will not attend if there is a 0/10 displayed because there seems to be no interest in that raid.
I guess the latter is worse than the former (as it can easily be discussed away).

TL;DR - the number is chosen this way because of a "lesser evil" decision and is meant to give an indicator if a raid will happen or not.

I'm currently modifying this value for "just list" raids, so I will have a look at this anyway.
Maybe there is a solution that is more explicit. Or maybe your proposal to switch between attended / set-up is a good alternative, too.

**Comment by LeonGeb** [comment]
I never looked at that this way, so thanks. I can at least through arguments to ppl who are asking.
I create another request that should go along with the attendance switch beautifully.
